packages/tpetra/core/src/Tpetra_Details_makeColMap_def.hpp

```diff
@@ -627,12 +627,16 @@ makeColMap (Teuchos::RCP<const Tpetra::Map<LO, GO, NT>>& colMap,
   //Pull down the sizes
   GO numLocalColGIDs = 0;
   GO numRemoteColGIDs = 0;
+  // DEEP_COPY REVIEW - DEVICE-TO-HOST
```

Comment:
These are deep_copy's of device views of scalars to host.

I'm at a loss for why we do this rather than use the parallel scan's return value capability.